src/Sylius/Bundle/AdminBundle/TwigComponent/Country/CountryType.php

```diff
@@ -38,11 +37,12 @@ final class CountryType
 
     public function __construct(
         private readonly FormFactoryInterface $formFactory,
+        private readonly string $formClass,
```

Comment:
Does it need something like `@param class-string $formClass`?

Reply:
We can add it.